libs/core/cornflow_core/schemas/query.py

```diff
@@ -11,3 +11,14 @@ class BaseQueryFilters(Schema):
     offset = fields.Int(required=False, dump_default=0)
     creation_date_gte = fields.DateTime(required=False)
     creation_date_lte = fields.DateTime(required=False)
+    id = fields.Str(required=True)
+    name = fields.Str(required=True)
+    deletion_date_gte = fields.DateTime(required=False)
+    deletion_date_lte = fields.DateTime(required=False)
+    update_date = fields.DateTime(required=False)
+    user_name = fields.Str(required=True)
+    last_name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    role_id = fields.Int(required=True)
+    url_rule = fields.Str(required=True)
+    description = fields.Str(required=True)
```

Comment:
Careful here, no query field should be `required=True` that can give a lot of problems. The four ones that we had are `required=False`